hpat/datatypes/hpat_pandas_series_functions.py

```diff
@@ -1096,6 +1096,57 @@ def hpat_pandas_series_ge_impl(self, other):
     raise TypingError('{} The object must be a pandas.series and argument must be a number. Given: {} and other: {}'.format(_func_name, self, other))
 
 
+@overload_method(SeriesType, 'idxmin')
+def hpat_pandas_series_idxmin(self, axis=None, skipna=True):
+    """
+    Pandas Series method :meth:`pandas.Series.idxmin` implementation.
+
+    .. only:: developer
+
+        Test: python -m hpat.runtests hpat.tests.test_series.TestSeries.test_series_idxmin1
+        Test: python -m hpat.runtests hpat.tests.test_series.TestSeries.test_series_idxmin_str
+        Test: python -m hpat.runtests hpat.tests.test_series.TestSeries.test_series_idxmin_int
+        Test: python -m hpat.runtests hpat.tests.test_series.TestSeries.test_series_idxmin_no
+
+    Parameters
+    -----------
+    axis :  :obj:`int`, default: None
+                *unsupported*
+    skipna:  :obj:`bool`, default: True
+                Exclude nan values when computing the result
+
+    Returns
+    -------
+    :obj:`pandas.Series` or :obj:`int` or :obj:`float`
```

Comment:
It looks like description is wrong. The method returns label of selected index. I'm not sure but it looks like it returns a value (corresponded type) of the index.

Reply:
Fixed